ts/index.ts

```diff
@@ -1,27 +1,40 @@
-// @ts-ignore
 import { NativeModules } from 'react-native'
 
-const { RNJolocom } = NativeModules;
+const { RNJolocom } = NativeModules
 
-export default RNJolocom;
-export * from './transports'
+export default RNJolocom
 export * from './storage'
 export * from './passwordStore'
 export * from './secureStorage'
 export * from '@jolocom/sdk'
 export * from 'jolocom-lib'
+
+/**
+ * Re-exporting available plugins
+ */
+export { DeepLinkingProvider } from './plugins/deep-linking/DeepLinkingProvider'
+export { JolocomDeepLinkingTransport } from './plugins/deep-linking/JolocomDeepLinkingTransport'
+export {
+  LinkingErrorCode,
+  JolocomLinking,
+} from './plugins/linking/JolocomLinking'
+export { JolocomWebSockets } from './plugins/web-sockets/JolocomWebSockets'
```

Comment:
I think the additional nesting doesn't bring any value here. Would be better if all the plugins lay directly under `/plugins`.